pkg/common/namespace.go

```diff
@@ -56,14 +57,29 @@ func updateNamespace(ns *apiV1.Namespace) error {
 	return err
 }
 
+func NamespaceHasLabels(namespace string, labels map[string]string) error {
+	ns, err := GetNamespace(namespace)
+	if err != nil {
+		return err
+	}
+	nsLabels := ns.GetLabels()
+	for k, v := range labels {
+		if nsLabels[k] != v {
```

Comment:
recommend using proper names. something like labelkey, labelvalue instead of k,v

Reply:
I think `k, v` is pretty straightforward and time saver.

Reply:
Ack! No issues.